src/BUILD.bazel

```diff
@@ -1,13 +1,14 @@
-load("@build_bazel_rules_nodejs//packages/typescript:index.bzl", "ts_library")
+load("@rules_typescript_proto_deps//@bazel/typescript:index.bzl", "ts_library")
 load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
 
 nodejs_binary(
     name = "change_import_style",
     data = [
         ":change_import_style_lib",
+        "@rules_typescript_proto_deps//:node_modules"
     ],
     entry_point = ":change_import_style.ts",
-    node_modules = "@rules_typescript_proto_deps//:node_modules",
+    templated_args = ["--bazel_patch_module_resolver"],
```

Comment:
Using `bazel_patch_module_resolver` seems like a temporary fix. Can we use the runfiles library so we don't have to fix this again in the future?

Reply:
nodejs_binary should be able to resolve this but for some reason it's not. @alexeagle encountered the same issue here: https://github.com/bazelbuild/rules_nodejs/blob/d5e84130f1c54f6149eae1ed9afb39c0a18670d3/packages/labs/grpc_web/BUILD.bazel#L18